### ISSUE TITLE
KeyError: 'acc'

### ISSUE THREAD

**Issue by profdrskipper** [mrtkp9993/DeepLearningExamples#1]
I get this error repeatedly around the 34th or 36th epoch over multiple executions tries. Not sure if this is a problem for other users as well. 

```
Traceback (most recent call last):
  File "classification.py", line 55, in <module>
    plt.plot(history.history['acc'])
KeyError: 'acc'

```

**Comment by mrtkp9993** [comment]
I far as I can see, different versions of Keras have different history dictionary keys.  You can add a print statement to see which keys that history have:

```python
print(history.history.keys()
```

The key probably will be "accuracy". So following code will work I think:

```
plt.plot(history.history['accuracy'])
plt.plot(history.history['val_accuracy'])
plt.title('Model Performance')
plt.ylabel('accuracy')
plt.xlabel('epoch')
plt.legend(['train', 'val'], loc='upper left')
plt.show()
```

**Comment by mrtkp9993** [comment]
Resolved with 217d0a8.
